controllers/apim/ratelimitpolicy_controller.go

```diff
@@ -496,6 +484,37 @@ func (r *RateLimitPolicyReconciler) reconcileLimits(ctx context.Context, rlp *ap
 	return nil
 }
 
+// fetchOperationsFromVS captures match rules from VirtualService and fill into RateLimitPolicy
+func fetchOperationsFromVS(vs *networkingv1alpha3.VirtualService, rlp *apimv1alpha1.RateLimitPolicySpec) {
```

Comment:
I do not get this. This method is building up a ratelimit policy out of the VS. But the reate limit policy is owned by the customer, so the controller is modifying it and it shouldn't.  I guess I am missing something

Reply:
It won't update the user-owned `RateLimitPolicy` on the cluster but instead, only the struct created from it that is passed to the wasm-shim.